main/src/modules/attentionRelated/ears/src/ears.cpp

```diff
@@ -19,13 +21,23 @@ bool ears::configure(yarp::os::ResourceFinder &rf)
         Time::delay(1.0);
     }
 
-    portToBehavior.open("/" + moduleName + "/behavior:o");
     portTarget.open("/" + moduleName + "/target:o");
     portToSpeechRecognizer.open("/" + moduleName + "/speech:o");
 
     MainGrammar = rf.findFileByName(rf.check("MainGrammar", Value("MainGrammar.xml")).toString());
     bShouldListen = true;
 
+    if (onPlannerMode){
+        port_planner.open("/" + moduleName + "/target:o");
```

Comment:
This has the same port name as `portTarget` .. seems like it really should be one variable :)